frontend/src/metabase/dashboard/components/DashCard/DashCardTabMenu/DashCardTabMenu.tsx

```diff
@@ -1,17 +1,70 @@
+import { t } from "ttag";
 import type { DashCardId } from "metabase-types/api";
+import type { StoreDashboardTab } from "metabase-types/store";
+import { Menu, Text } from "metabase/ui";
+import Tooltip from "metabase/core/components/Tooltip";
 import { useDashCardTabMenu } from "./use-dash-card-tab-menu";
+import {
+  TabButton,
+  VerticalDivider,
+  ChevronStyledIcon,
+} from "./DashCardTabMenu.styled";
 
 interface DashCardTabMenuProps {
   dashCardId: DashCardId;
 }
 
 export function DashCardTabMenu({ dashCardId }: DashCardTabMenuProps) {
   const { showMenu, tabs, moveToTab } = useDashCardTabMenu(dashCardId);
+  const [suggestedTab] = tabs;
 
   if (!showMenu) {
     return null;
   }
 
-  // TODO menu html + styles
-  return <a onClick={() => moveToTab(tabs[1].id)}>Move to {tabs[1].name}</a>;
+  return (
+    <>
+      <Text color="bg-dark" size="sm" ml={5}>
+        {t`Move to `}
+        <Tooltip tooltip={t`Move to ${suggestedTab.name} tab`}>
+          <TabButton size="sm" onClick={() => moveToTab(suggestedTab.id)}>
+            {suggestedTab.name}
+          </TabButton>
+        </Tooltip>
+      </Text>
+
+      {tabs.length > 1 && (
+        <TabChevronMenu tabs={tabs} onTabSelect={tabId => moveToTab(tabId)} />
+      )}
+
+      <VerticalDivider />
+    </>
+  );
+}
+
+type TabChevronMenuProps = {
```

Comment:
Personally I would split this out into a different file to keep one file <> one component